docs/requirements.md

```diff
@@ -15,4 +15,14 @@ You can stil use `periodic`, but it is not recommended unless you know exactly w
 
 For the second, Scalar DB does not work on some Cassandra compatible databases such as [Amazon Keyspaces](https://aws.amazon.com/keyspaces/) that don't support LWT since `ConsensusCommit` transaction manager relies on linearizable operations of underlining databases to make transactions serializable. 
 
-If the above requirements are met, storage operations with `LINEARIZABLE` can provide linearizablity and transaction operations with `SERIALIZABLE` can provide strict serializability.
\ No newline at end of file
+If the above requirements are met, storage operations with `LINEARIZABLE` can provide linearizablity and transaction operations with `SERIALIZABLE` can provide strict serializability.
+
+## JDBC databases
+
+In Scalar DB on JDBC databases, you can't choose a consistency level (`LINEARIZABLE`, `SEQUENTIAL` or `EVENTUAL`) in your code with the `Operation.withConsistency()` method, and the consistency level depends on the setup of your JDBC database.
+For example, when you have asynchronous read replicas in your setup and perform read operations against them, the consistency level will be `EVENTUAL`, because you can read stale data from the read replicas.
```

Comment:
```suggestion
For example, when you have asynchronous read replicas in your setup and perform read operations against them, the consistency will be eventual because you can read stale data from the read replicas.
```
Suggested because the original description sounds like the consistency level is set to `EVENTUAL`.